recipe/meta.yaml

```diff
@@ -26,14 +26,17 @@ requirements:
 
   run:
     - libgfortran
+    - libgcc
```

Comment:
Would recommend doing the following if we are adding `libgcc`.

```yaml
    - libgfortran  # [osx]
    - libgcc       # [linux]
```

Unfortunately it has generally been a pain to have `libgcc` as a `run` dependency on macOS. Particularly for low level packages like `matplotlib`, `numpy`, and `scipy` as examples.